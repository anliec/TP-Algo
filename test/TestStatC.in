STATS_C 2

ADD 2 2015 06 23 21 26 1 V
ADD 2 2015 06 23 21 25 1 V
STATS_C 2
ADD 2 2015 06 23 21 24 1 J
STATS_C 2
ADD 2 2015 06 23 21 23 1 R
STATS_C 2
ADD 2 2015 06 23 21 22 1 N
STATS_C 2

ADD 2 2015 06 23 21 26 1 V
STATS_C 2
ADD 2 2015 06 23 21 26 1 J
STATS_C 2
EXIT
