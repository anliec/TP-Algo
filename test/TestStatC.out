V 0%
J 0%
R 0%
N 0%
V 100%
J 0%
R 0%
N 0%
V 66%
J 33%
R 0%
N 0%
V 50%
J 25%
R 25%
N 0%
V 40%
J 20%
R 20%
N 20%
V 50%
J 16%
R 16%
N 16%
V 42%
J 28%
R 14%
N 14%
